roles/puma-svc/templates/sudoers.d.j2

```diff
@@ -0,0 +1,5 @@
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl stop app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl start app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl restart app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl status app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/journalctl
```

Comment:
No, we were intending broad journalctl access (we have been granting that fairly widely). I'm sure there are some dangerous options there, but generally we trust our devs not to intentionally falsify or purge logs...